orangecontrib/text/vectorization/base.py

```diff
@@ -0,0 +1,11 @@
+class BaseVectorizer:
+    """Base class for vectorization objects. """
+    name = NotImplemented
+
+    def transform(self, corpus):
```

Comment:
Absolutely not! If anything, `__call__` can wrap `transform()`. Scikit defined this really nicely, and it's become an industry standard one ought to like to intend to upkeep.